extensions/filesystem/configuration-fs/src/main/java/org/eclipse/dataspaceconnector/configuration/fs/FsConfigurationExtension.java

```diff
@@ -69,6 +69,7 @@ public void initialize(Monitor monitor) {
             properties.load(is);
             for (String name : properties.stringPropertyNames()) {
                 propertyCache.put(name, properties.getProperty(name));
+                System.setProperty(name, properties.getProperty(name));
```

Comment:
Why is this necessary? seems like a hack.

Reply:
This was necessary because the FsConfigurationExtension didn't work with the `propOrEnv` method. But as this is not necessary for this PR anymore, I removed the fix.